codewof/__init__.py

```diff
@@ -0,0 +1 @@
+"""Init file for codeWOF."""
```

Comment:
(Putting nowhere in particular)

![image](https://user-images.githubusercontent.com/25916475/65398561-108a8c80-de0c-11e9-95eb-803cdf718068.png)

It would be nice to get a message "earned X points" or something when you solve something so you get alerted when you earn points